packages/template-retail-react-app/app/hooks/use-wish-list.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {useCustomerProductLists, useShopperCustomersMutation} from 'commerce-sdk-react-preview'
+import {useCurrentCustomer} from './use-current-customer'
+
+export const useWishList = ({listId = ''} = {}) => {
```

Comment:
I don't feel strongly, but I feel like this `listId` option is gonna be very very rarely used in the context of retail react app and seems like we are overthinking.